### PR TITLE
Clarification of voting rules under 1.1.2.c

### PR DESCRIPTION
**Pull request by TRManderson**
This allows people who would be lawbreakers under the would-be policy time to change their ways before punishments can be enacted upon then.

See #3 for discussion of the existing constitution section if clarification is needed.

### CODE REVIEW

File: Constitution.md [hunk 1]
```diff
@@ -32,8 +32,7 @@
 
 1.1.2.c A pull request will remain unresolved until the next session of the ruling party, at least five days from the time of proposal.
 
-1.1.2.c.a If a pull request achieves affirmative votes from more than half the ruling body as a whole, it will immediately be ratified.
+1.1.2.c.a If a pull request has affirmative votes from more than half the ruling body as a whole at a session of the ruling party, it will immediately be ratified. A :+1: reaction to the pull request is considered an affirmative vote.
 
-1.1.2.c.b If a pull request achieves negative votes from more than half the ruling body as a whole, it will immediately be closed with no effect.
-
-1.1.2.d A pull request will be ratified if a majority of votes (greater than half) on the pull request are in the affirmative.
+1.1.2.c.b If a pull request has negative votes from more than half the ruling body as a whole at a session of the ruling party, it will immediately be closed with no effect. A :-1: reaction to the pull request is considered an affirmative vote
```
Comment (Guthers) [inline Constitution.md]: ":-1: reaction to the pull request is considered an affirmative vote"

Do you mean a negative vote?

**Comment by TRManderson** [comment]
Yes. I'll also be revising this further

**Comment by TRManderson** [comment]
This has been revised.

tl;dr:
If a majority of the ruling party shares a position on a PR, it is resolved at the next session of the ruling party. If 2/3 of the ruling party share a position on a PR, it is resolved immediately.

**Comment by CameronAavik** [comment]
I would like to suggest a change to be made before accepting this. All votes made on a pull request are nullified if a revision is made before being accepted. A new post must be made in the pull request and votes must start from that post from scratch.

This is in the best interest of everybody in case revisions are made which we do not agree with.
